cc

```diff
@@ -141,10 +141,11 @@ while (( $# )); do
 	fi
 	shift
 done
-if [ "$set_flags" = 0 ]; then
-	echo /usr/local/go/bin/go $SUBCMD -ldflags "-s -w ${LDFLAGS:-}" "${arr[@]}"
-	exec /usr/local/go/bin/go $SUBCMD -ldflags "-s -w ${LDFLAGS:-}" "${arr[@]}"
-else
-	echo /usr/local/go/bin/go $SUBCMD "${arr[@]}"
-	exec /usr/local/go/bin/go $SUBCMD "${arr[@]}"
+
+if [ "$set_flags" = 0 ] && [ -n "$LDFLAGS" ]; then
```

Comment:
Do we even need the ldflags handling logic? We can pass to the compiler whatever was passed in from the call.

Reply:
This logic is for supporting $LDFLAGS from environment variables. When both flags are and env variable is set, they should be merged.

Do you know any LDFLAGS environment variable which is supported by `go` out of the box?

Reply:
`go help environment` does not have a specific flags for the linker, however it's possible to use `GOFLAGS=-ldflags=-w`.

But, yeah, for some reason I remembered that go also used LDFLAGS. Anyways, yeah, then it's necessary at the moment -- however, we should probably migrate to not using LDFLAGS when calling this docker image and then dropping this special LDFLAGS altogether.

In that case, the change is fine as is.